crypto/armor.go

```diff
@@ -171,7 +190,7 @@ func UnarmorDecryptPrivKey(armorStr string, passphrase string) (privKey cryptoty
 		return privKey, "", fmt.Errorf("unrecognized armor type: %v", blockType)
 	}
 
-	if header["kdf"] != "bcrypt" {
+	if header["kdf"] != KDFBcrypt && header["kdf"] != KDFArgon2 {
```

Comment:
Maybe define this `"kdf"` as a constant?. There are couple of places where the string literal is used.